app/src/pages/common/settingsPage/autoAnalysisTab/analysisForm/strategyBlock/strategyBlock.jsx

```diff
@@ -70,6 +90,8 @@ export class StrategyBlock extends Component {
     intl: PropTypes.object.isRequired,
     disabled: PropTypes.bool,
     data: PropTypes.object,
+    uniqueErrorAutoAnalysisDisabled: PropTypes.bool,
+    uniqueErrorsRemoveNumbersDisabled: PropTypes.bool,
```

Comment:
Where from it passed? Seems to be removed.

Reply:
agree